license_solver/__init__.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+# solver-license-job
+# Copyright(C) 2021 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""init package."""
+
+from .license_solver import LicenseSolver
+
+# from .licenses import *
+# from .classifiers import *
+# from .json_solver import *
+# from .comparator import *
+
+
+__author__ = "Viliam Podhajecky <vpodhaje@redhat.com>"
+
+__all__ = ["LicenseSolver"]
```

Comment:
You can also add `__version__` here, bots will automatically maintain the version identifier.